### ISSUE TITLE
HP Bar

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#4]
```
What steps will reproduce the problem?
1. Start in portrait mode
2. Switch to landscape
3. Switch back to portrait

What is the expected output? What do you see instead?
Even though HP is full and in the initial portrait mode th HP bar was full 
switching to landscape shows the HP bar as a fraction of full even though the 
actual HP is full.

What version of the product are you using? On what operating system?
0.6.4 on Android 2.2

Please provide any additional information below.
Not sure if this issue also affects the Level Bar as I have no Level points.

```

Original issue reported on code.google.com by `The2ndclory` on 19 Nov 2010 at 1:27

**Comment by GoogleCodeExporter** [comment]
```
Thanks for the report. 

I've confirmed that this is a bug. It even affects all progressbars in game, 
even those on the character screen. However, it's purely cosmetical, but 
annoying when playing.

Some google-fu reveals that setting the progressdrawable in xml rather than in 
code should resolve the issue. I'll try that shortly. 

In the meantime, I'll decrease the issue priority to "low" since it's only 
cosmetical. Please let me know if you disagree.
```

Original comment by `oskar.wi...@gmail.com` on 20 Nov 2010 at 9:35
- Added labels: **Priority-Low**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
Thats perfectly fine as long as youre aware.
```

Original comment by `The2ndclory` on 20 Nov 2010 at 1:03

**Comment by GoogleCodeExporter** [comment]
```
This one is fixed.
```

Original comment by `SamuelPl...@gmail.com` on 26 Nov 2010 at 10:23

**Comment by GoogleCodeExporter** [comment]
```
Perhaps its not fixed.
It works well for my own hp bar. But it happens again lower monsters. If I:
1) attack some easy monsters
2) switch to landscape and back
3) attack the same easy monsters several times it
4) the hp bar in portrait shows only a fraction of the real hp.
```

Original comment by `SamuelPl...@gmail.com` on 26 Nov 2010 at 11:41

**Comment by GoogleCodeExporter** [comment]
```
version 0.6.6 (I assume; how to make sure?)
android 2.1 (huawei 8110 / T-Mobil pulse)
For me hp bar is ok, but some monster bars are strange (though I don't turn to 
landscape)
e.g. 3/3 shown as 50%(!) but 1/3 is already shown ok.
```

Original comment by `surrano` on 8 Dec 2010 at 10:49

**Comment by GoogleCodeExporter** [comment]
```
Yes the bug is still present in the newest version.
```

Original comment by `SamuelPl...@gmail.com` on 8 Dec 2010 at 11:40

**Comment by GoogleCodeExporter** [comment]
```
I get the cosmetic HP bar problem in portrait mode, without having been in 
landscape mode.
```

Original comment by `kims...@gmail.com` on 18 Dec 2010 at 10:50

**Comment by GoogleCodeExporter** [comment]
```
The screenshots for a possible quickitems patch, issue 6, shows this problem 
still exist and I believe it to draw heal according to 100 HP instead of the 
actual maximum HP for the monster. That might be a clue to fix this issue.
```

Original comment by `kims...@gmail.com` on 10 May 2011 at 5:28

**Comment by GoogleCodeExporter** [comment]
```
This bug might have been fixed in v0.6.9 , since it contained some changes to 
the layout of progressbars.

I am not able to reproduce it anymore. Should we close it?
```

Original comment by `oskar.wi...@gmail.com` on 8 Jul 2011 at 12:51

**Comment by GoogleCodeExporter** [comment]
```
Tried hard to reproduce, but I couldn't.
IMO close it.
```

Original comment by `SamuelPl...@gmail.com` on 9 Jul 2011 at 4:11

**Comment by GoogleCodeExporter** [comment]
```
Thanks for re-testing.

Closing, fixed in v0.6.9 .
```

Original comment by `oskar.wi...@gmail.com` on 13 Jul 2011 at 8:30
- Changed state: **Fixed**
- Added labels: **Milestone-0.6.9**
